### PR TITLE
feat(windows-jdk17): maven-17-windows

### PR DESCRIPTION
**Pull request by smerle33**
as per https://github.com/jenkins-infra/helpdesk/issues/2822
this should provide an image for maven with jdk17 on windows

based on jenkins/inbound-agent:jdk17-nanoserver-1809 ([https://hub.docker.com/layers/jenkins/inbound-agent/jdk17-nanoserver-1809/images/sha[…]b8cbef65b63c606ff59ef1554059e824d047abeda3258?context=explore](https://hub.docker.com/layers/jenkins/inbound-agent/jdk17-nanoserver-1809/images/sha256-5303f9f758cd8282ac4b8cbef65b63c606ff59ef1554059e824d047abeda3258?context=explore))

### CODE REVIEW

File: maven/jdk17/Dockerfile.nanoserver [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# escape=`
+ARG JAVA_VERSION=11.0.17_8
```
Comment (lemeurherve) [inline maven/jdk17/Dockerfile.nanoserver:2]: Shouldn't all these java & jdk versions be 17.x in this file?

Comment (dduportal) [reply maven/jdk17/Dockerfile.nanoserver:2]: We need both:

- JDK17 from the inbound image, as the default JDK17 used by tthe builds (variable `JAVA_HOME` + binary `java` in the Path).
- JDK11 on a fixed location to allow execution of the agent process (that should be the same JDK as the controller).

Comment (lemeurherve) [reply maven/jdk17/Dockerfile.nanoserver:2]: Oh right, sorry my mistake, nevermind my comment.

**Review by dduportal — APPROVED** [review]
LGTM !

**Comment by dduportal** [comment]
Merging: the check is ok (the failure is updatecli diff because of new manifest)

**Pull request merged**